src/main/scala/consul/v1/catalog/CatalogRequests.scala

```diff
@@ -52,7 +51,9 @@ object CatalogRequests {
     Json.writes[Registerable]
   }
 
-  def apply(basePath: String)(implicit executionContext: ExecutionContext): CatalogRequests = new CatalogRequests {
+  def apply(basePath: String)(implicit executionContext: ExecutionContext, rb: ConsulRequestBasics): CatalogRequests = new CatalogRequests {
+
+    import rb._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/Codacy/scala-consul/file/1255759076/issues/source?bid=2788492&fileBranchId=2939115#l56)